src/main/scala/mesosphere/marathon/core/task/update/TaskStatusUpdateConfig.scala

```diff
@@ -0,0 +1,22 @@
+package mesosphere.marathon.core.task.update
+
+import org.rogach.scallop.ScallopConf
+
+trait TaskStatusUpdateConfig extends ScallopConf {
+  //scalastyle:off magic.number
+
+  lazy val internalMaxParallelStatusUpdates = opt[Int]("max_parallel_status_updates",
+    descr = "INTERNAL TUNING PARAMETER: The maximum number of status updates that get processed in parallel.",
+    noshort = true,
+    hidden = true,
+    default = Some(20)
```

Comment:
Testing. I used something bigger before and it was not a success. I think `20` is erring on the side of non-optimal performance but no havoc.